transport.go

```diff
@@ -72,7 +74,7 @@ func (t *pipeTransport) Send(msg map[string]interface{}) error {
 	if os.Getenv("DEBUGP") != "" {
 		fmt.Fprint(os.Stdout, "\x1b[32mSEND>\x1b[0m\n")
 		if err := json.NewEncoder(os.Stdout).Encode(msg); err != nil {
-			logger.Printf("could not encode json: %v\n", err)
+			logger.Error("could not encode json", pwlogger.ErrAttr(err))
```

Comment:
Only for debugging when developing playwright-go. No errors are printed in the new PR.      https://github.com/playwright-community/playwright-go/pull/500/files#diff-5982a4ca350449c8a5deba675bc4bc4bb0f1823c45e6c7ba0076cc6b159fef8fR73